pkgs/crypto/crypto.go

```diff
@@ -620,7 +620,7 @@ func GetPubCommitsFromSharesData(reshare *wire.Reshare) ([]kyber.Point, error) {
 			return nil, err
 		}
 		kyberPubhare := &share.PubShare{
-			I: int(i),
+			I: int(reshare.OldOperators[i].ID - 1),
```

Comment:
how can we guarantee that the order of the pubkeys matches the order of the operators in the reshare message?